docs/products/clickhouse/howto/clickhouse-query-cache.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Use query cache in Aiven for ClickHouse®
+sidebar_label: Use query cache
+---
+
+Aiven for ClickHouse® provides a query cache mechanism that helps improve query performance
+by caching query results.
+
+## How it works
+
+When the Aiven for
+[ClickHouse query cache](https://clickhouse.com/docs/en/operations/query-cache) is enabled,
+multiple identical `SELECT` queries run simultaneously are computed only once,
+and further executions of the same query are served directly from the cache.
+
+:::important
+By default, the Aiven for ClickHouse query cache is disabled for all `SELECT` queries.
+:::
+
+## Why use it
+
+Using query cache in your Aiven for ClickHouse services can help reduce latency and
+resource consumption.
+
+Key use case for the Aiven for ClickHouse query cache are the following:
+
+- Performance enhancement: Reducing latency and load for frequently executed queries
+- Analytical workloads: Using complex repetitive queries that involve high data aggregation
+
+## Enable query cache
+
+To enable the query cache for a query, use an SQL client (for example, the
```

Comment:
It still sounds like using client and applying settings in this way is necessary.
I would white something like
"To enable the query cache, setting `use_query_cache` should be set to `1`. It can be done by adding `SETTINGS use_query_cache = 1` to the end of your query.